### DISCUSSION TITLE
Language learning video player and reader with dictionary lookups powered by Isar

### DISCUSSION THREAD

**Discussion by arianneorpilla** [isar/isar#651] [Show and tell]
Hi everyone, I just want to share my Flutter application that I've recently released a [**new version**](https://github.com/lrorpilla/jidoujisho/releases/tag/2.0.0) and migrated to using Isar with, [**jidoujisho**](https://github.com/lrorpilla/jidoujisho).

It's an app for Japanese language learning that allows you to make dictionary lookups and export flashcards as you watch videos and e-books.

I switched over from ObjectBox to Isar, going from searching a single dictionary to querying multiple dictionaries at once. I'm really, really happy with Isar, it's been a blast developing with and making use of it and I just wanted to share my work.

I'm using 3.0.0-dev.0 as I started developing with 2.2.1 and when pushing a build for my latest release, I needed to support 32-bit Android users. I really want to get that extra performance from put operations that I've read are in the latest 3.0 change logs ([some of my users are experiencing crashes importing over 300K objects on their tablet](https://github.com/lrorpilla/jidoujisho/issues/113)), but I can't fully upgrade to that yet since I use type converters and I'm not sure how to migrate to enumerated objects, so really looking forward to those docs and a stable 3.0 Isar release.

This package is great. Thank you for making it!

**Comment by muddassar-lab** [comment]
Isar Database developer said :
@leisim said: "Soon 😅 sorry no exact date but very soon. Just a few small fixes, inspector support for the newest features and web support" [Source: 30/08/2022 @ Telegram](https://t.me/isardb/2766)

**Comment by simc** [comment]
Wow looks great thanks a lot for sharing!! ❤️
